src/primitives/orchard.h

```diff
@@ -108,6 +113,18 @@ class OrchardBundle
             return std::nullopt;
         }
     }
+
+    bool OutputsEnabled() const {
+        return orchard_bundle_outputs_enabled(inner.get());
+    }
+
+    bool SpendsEnabled() const {
+        return orchard_bundle_spends_enabled(inner.get());
+    }
+
+    bool HasValidCoinbaseOutputs() const {
```

Comment:
This returns false if there is no Orchard bundle, but shouldn't it vacuously return true in that case because all Orchard coinbase outputs are valid? In any case, ensure that `orchard_bundle_has_valid_coinbase_outputs` has the desired behaviour for the no-bundle case and drop the `inner &&`.

Reply:
Ah, good catch.